docker/images/pinot/etc/jmx_prometheus_javaagent/configs/broker.yml

```diff
@@ -157,15 +157,37 @@ rules:
 
   ## Metrics that fit the catch-all patterns above should not be added to this file.
   ## In case a metric does not fit the catch-all patterns, add them before this comment
-
-  # This is a catch-all pattern for pinot table metrics with offline/realtime suffix.
+  # This is a catch-all pattern for pinot table metrics with offline/realtime suffix without kafka topic
   # Patterns after this line may be skipped.
```

Comment:
nit: I feel that we should add this comment for each catch-all pattern so that people know the potential risks?